flow/packet-headers/ipv6_routing.yaml

```diff
@@ -0,0 +1,373 @@
+        segment_routing:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting'
+          x-field-uid: 2
+    Flow.Ipv6SegmentRouting:
+      description: Defines the structure of the IPv6 Segment Routing Header (SRH)
+        with Routing Type 4. This header is an IPv6 Routing header used to specify
+        a source-routed path for a packet, guiding it through a sequence of Segment
+        IDs (SIDs), which are typically IPv6 addresses.
+      type: object
+      properties:
+        segments_left:
+          x-field-pattern:
+            description: 8-bit unsigned integer containing the number of remaining
+              segments to be visited before the packet reaches its final destination.
+              It is decremented at each segment endpoint. It points to the current
+              active segment in the Segment List. This should not be more than the
+              number of segment specified in the segment list. When auto is assigned 
+              the value is set to the number of segment specified.
+            format: integer
+            length: 8
+            default: 0
+            features: [auto, count]
+          x-field-uid: 1
+        last_entry:
+          x-field-pattern:
+            description: 8-bit unsigned integer that contains the zero-based index
+              of the last element in the Segment List array. For example, if the Segment
+              List contains 3 addresses (at indices 0, 1, 2), the value of Last Entry
+              is 2. When auto is assigned the value is set to one less than the number 
+              of segment specified.
+            format: integer
+            length: 8
+            default: 0
+            features: [auto, count]
+          x-field-uid: 2
+        flags:
+          $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.Flags'
+          x-field-uid: 3
+        tag:
+          x-field-pattern:
+            description: 16-bit field used to tag a packet as part of a class or group,
+              enabling shared properties or policy treatment. If not used, it MUST
+              be set to zero.
+            format: integer
+            length: 16
+            default: 0
+            features:
+              - count
+          x-field-uid: 4
+        segment_list:
+          description: 128-bit IPv6 addresses representing the nth segment in the
+            Segment List. The Segment List is encoded starting from the last segment
+            of the SR Policy. That is, the first element of the Segment List (Segment List[0])
+            contains the last segment of the SR Policy, the second element contains
+            the penultimate segment of the SR Policy, and so on.
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.Segment'
+          x-field-uid: 5
+        tlvs:
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.Ipv6SegmentRouting.Tlvs'
```

Comment:
I feel class name could be `Flow.Ipv6SegmentRouting.Tlv` no `s`. 
Note: `tlvs` is absolutely fine. SO, no need to change the property name.